types/rc-tooltip/rc-tooltip-tests.tsx

```diff
@@ -56,5 +56,5 @@ ReactDOM.render(
 const props: RCTooltip.Props = {
     placement: "bottomRight",
     trigger: ['click', 'focus'],
-    overlay: () => <span>tooltip</span>,
+    overlay: <span>tooltip</span>,
```

Comment:
This modification is not correct since `overlay` accepts both React component and a function which returns React component.

https://www.npmjs.com/package/rc-tooltip

Previous test code should work.

Instead of the modifications to `rc-tooltip-tests.tsx`, could you apply following patch?

```diff
diff --git a/types/rc-tooltip/index.d.ts b/types/rc-tooltip/index.d.ts
index 78a1fb63fb..97b77bb39f 100644
--- a/types/rc-tooltip/index.d.ts
+++ b/types/rc-tooltip/index.d.ts
@@ -31,7 +31,7 @@ declare namespace RCTooltip {
 		placement?: Placement | Object;
 		align?: Object;
 		onPopupAlign?: (popupDomNode: Element, align: Object) => void;
-		overlay: React.ReactNode;
+		overlay: React.ReactNode | (() => React.ReactNode);
 		arrowContent?: React.ReactNode;
 		getTooltipContainer?: () => Element;
 		destroyTooltipOnHide?: boolean;
```

Reply:
Awesome. Thanks for getting back to us. Fine example of the overly permissive children before making liars if us all :)

Will follow up later today. Thanks again